.appveyor.yml

```diff
@@ -13,3 +13,7 @@ install:
   - nuget restore proj/vs2017
 build:
   project: proj/vs2017/NAS2D.sln
+test_script:
+  - cd %APPVEYOR_BUILD_FOLDER%\test
+  - '%APPVEYOR_BUILD_FOLDER%\%PLATFORM%\%CONFIGURATION%\test.exe'
```

Comment:
I was using Windows specific directory separators here since it builds for Windows. Just realized the rest of the file had Linux style directory separators. Could be cleaned up. Looks like either separator works.

Reply:
In this case it makes sense to use "\" since these are shell commands, and so need to use the Windows specific "\". The other paths are not shell commands, nor are they used to run shell commands. They are likely fed directly to the kernel through the filesystem API, and so will accept either directory separator.

Reply:
Guess I'm good with whatever, just seemed strange.